src/main/java/kr/codesqaud/cafe/interceptor/CafeInterceptor.java

```diff
@@ -0,0 +1,26 @@
+package kr.codesqaud.cafe.interceptor;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.web.servlet.HandlerInterceptor;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+
+public class CafeInterceptor implements HandlerInterceptor {
```

Comment:
```suggestion
public class LogInIntercepter implements HandlerInterceptor {
```
카페 인터셉터보다 조금 구체적인 이름이 좋을 것 같아요.